package review12;

public class Exception1 {
    public static void main(String[] args) {
        int number=10;
        int [] arr=new int[2];
        arr[5]=10;
    }
    }
